libafl/src/state/mod.rs

```diff
@@ -787,6 +871,10 @@ where
         let mut state = Self {
             rand,
             executions: 0,
+            pending: 0,
```

Comment:
can you add this member in stages/stats.rs?

Reply:
instead of putting in state

Reply:
how about adding the "pending" " pend_favor" "own_finds" to stages/stats.rs and remaining the "imported" in state?
I found it not easy to calculate the value of the "imported" in stats.rs. @tokatoka